res/values/strings.xml

```diff
@@ -465,6 +465,7 @@
     <string name="confirmation_remove_files_alert">Do you really want to remove the selected items?</string>
     <string name="confirmation_remove_folders_alert">Do you really want to remove the selected items and their contents?</string>
     <string name="maintenance_mode">Server in maintenance mode</string>
+    <string name="failed_upload_quota_exceeded_text">Not enough free space</string>
```

Comment:
@davigonz, check if that message fits in a single line in the uploads view. I have a bad feeling about it. 

iOS displays the message in a single line, we show more information in the same line. 

Could try to equalize the items in both platforms, but I wouldn't like this grows too much in UI changes.